src/modules-checker.ts

```diff
@@ -26,8 +27,8 @@ export class ModulesChecker {
     this.logger = new Logger(config)
   }
 
-  public checkModules(): string[] {
-    return this.parseDeps().es6Modules
```

Comment:
@tooppaaa I changed this to expose the whole result set based on what was in the README when I made this PR, but I am not sure if we should keep it as is now that people are using it (See the comments in #41). What do you think?

Reply:
I released only 3 days ago, maybe we should make it right as soon as possible.
I agree with your change as it's way more flexible in the future to avoid breaking changes.